src/Datadog.Trace/Logging/DatadogLogging.cs

```diff
@@ -106,20 +111,55 @@ static DatadogLogging()
 
                 SharedLogger = loggerConfiguration.CreateLogger();
 
+                // Use to immediately execute any startup logs
+                Initialized = true;
+
                 // Log some information to correspond with the app domain
-                SharedLogger.Debug(
-                    "OSArchitecture: {0}, OSDescription: {1}, ProcessArchitecture: {2}, FrameworkDescription: {3}",
+                SharedLogger.Information(
+                    "OSArchitecture: {0}, OSDescription: {1}, ProcessArchitecture: {2}, FrameworkDescription: {3}, CLR: {4}",
                     RuntimeInformation.OSArchitecture,
                     RuntimeInformation.OSDescription,
                     RuntimeInformation.ProcessArchitecture,
-                    RuntimeInformation.FrameworkDescription);
+                    RuntimeInformation.FrameworkDescription,
```

Comment:
I commented on this before: we should use `Datadog.Trace.FrameworkDescription` to get correct version numbers in .NET Core. The value returned by `RuntimeInformation.FrameworkDescription` is useless before .NET Core 3.0.